src/main/kotlin/com/project/goms/domain/outing/entity/repository/OutingRepository.kt

```diff
@@ -10,5 +10,6 @@ interface OutingRepository: CrudRepository<Outing, UUID> {
     fun existsByAccount(account: Account): Boolean
     fun queryAllByOrderByCreatedTimeDesc(): List<Outing>
     fun deleteByAccountIdx(accountIdx: UUID)
+    fun findByAccount_NameContaining(name: String): List<Outing>
```

Comment:
account name을 스네이크 하신 이유가 있으실까요? 전반적으로 코드들이 카멜케이스를 사용해서 카멜케이스로 바꾸는게 좋아보입니다.
그리고 find 보다는 위에 보면 query를 사용해서 조회 쿼리 메서드를 작성이 되어있는데 컨벤션을 맞추면 좋을거 같습니다.
```suggestion
    fun queryByAccountNameContaining(name: String): List<Outing>
```

Reply:
추가적으로 outing entity를 리스트로 반환 받는데 account entity는 지연로딩으로 되어있어, account entity에 관한 이름, 학번을 조회하게 된다면 쿼리가 추가적으로 발생하는거라 n + 1 문제가 발생하게 됩니다. 추후 fetch join을 사용하여 수정하는게 좋아보입니다!
하지만 지금은 기능 구현할 시간이 많이 없는지라 추후 나중에 해결하면 될거 같습니다.